src/playground/sample_data_tooling/sample_data_ingestor/README.md

```diff
@@ -0,0 +1,127 @@
+# Sample Data Tooling: Ingestion
+
+## Pre-requisites:
+
+Before using this tool, make sure the required libraries as listed in `requirements.txt` are downloaded. Because this tool requires the `OpenSearch Python client`, OpenSearch (OS) and OpenSearch Dashboards (OSD) must already be configured. Installations for OS can be found [here](https://opensearch.org/docs/latest/opensearch/install/index/) while installations for OSD can be found [here](https://opensearch.org/docs/latest/dashboards/install/index/). 
+
```

Comment:
Can you add link for the OpenSearch Python Client?

Reply:
Done, link added